Makefile.checker

```diff
@@ -34,22 +34,8 @@ CHECKMLLIBFILE := checker/.mllibfiles
 CHECKERDEPS := $(addsuffix .d, $(CHECKMLDFILE) $(CHECKMLLIBFILE))
 -include $(CHECKERDEPS)
 
-# Copied files
-checker/esubst.mli: kernel/esubst.mli
-	cp -a $< $@
-	sed -i.bak '1i(* AUTOGENERATED FILE: DO NOT EDIT *)\n\n\n\n\n\n\n\n' $@ && rm $@.bak
-checker/esubst.ml: kernel/esubst.ml
-	cp -a $< $@
-	sed -i.bak '1i(* AUTOGENERATED FILE: DO NOT EDIT *)\n\n\n\n\n\n\n\n' $@ && rm $@.bak
-checker/names.mli: kernel/names.mli
-	cp -a $< $@
-	sed -i.bak '1i(* AUTOGENERATED FILE: DO NOT EDIT *)\n\n\n\n\n\n\n\n' $@ && rm $@.bak
-checker/names.ml: kernel/names.ml
-	cp -a $< $@
-	sed -i.bak '1i(* AUTOGENERATED FILE: DO NOT EDIT *)\n\n\n\n\n\n\n\n' $@ && rm $@.bak
-
 ifeq ($(BEST),opt)
-$(CHICKEN): config/config.cmxa checker/check.cmxa checker/coqchk.mli checker/coqchk.ml
+$(CHICKEN): $(CORECMA:.cma=.cmxa) checker/check.cmxa checker/coqchk.mli checker/coqchk.ml
```

Comment:
Why are we linking all CORECMA here?

Reply:
Indeed, this is a leftover of the original PR that used many files around the codebase. This should be able to be changed to `kernel.cma` now. @ppedrot wait until this is fixed please.

Reply:
Fixed.